packages/components/src/typography.scss

```diff
@@ -0,0 +1,5 @@
+
+$aui-fontFamily-sans: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Oxygen-Sans', 'Ubuntu', 'Cantarell',
```

Comment:
I've pulled these out as variables since they'll likely be used across a number of the components.

Using a prefix to minimise the chance of clashing with variables already in use by the consuming developer.